### PR TITLE
Create Discover Members

### LINKED ISSUE #6: Create Discover Members
IMG-20240919-WA0049

### PR DESCRIPTION
**Pull request by realsandaru**
### Here I have created the Components related to the _**Account card**_ and the view of the _**Cards on the dashboard**_.

A **separate folder** is created for the Pngs that will be useful for other members of the team. Additionally, the width of the Final **"DiscoverMembers"** component has been set to 100% without a **'max-w'** bound allowing for more flexibility when integrating the component into the larger layout.

@RandilaP  I have included the **tailwind** dependencies in the package and the index.css has been setup to use tailwind.

### [x] Tasks assigned by the Issue #6 has been completed @Imashi-S 

Fixes #6

### CODE REVIEW

File: src/DiscoverMembers.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+              </div>
+              <div>
+                <Account label1="G Perera"/>
+              </div>
+              <div>
+                <Account label1="H Perera"/>
+              </div>
+              <div>
+                <Account label1="I Perera"/>
+              </div>
```
Comment (RandilaP) [inline src/DiscoverMembers.js]: Add a list of these names and map it with account card

**Review by RandilaP — CHANGES_REQUESTED** [review]

File: src/DiscoverMembers.js (hunk 1 above)
Comment (realsandaru) [reply src/DiscoverMembers.js]: Done

**Comment by realsandaru** [comment]
DIscoverMembers

1a454462-808d-4401-b3db-d77afbb8e0c3

SearchBar

29008d4a-f778-4182-a411-8d8f9573612e

**Comment by RandilaP** [comment]
good job @realsandaru

**Pull request merged**